bec_alerts/models.py

```diff
@@ -20,6 +20,31 @@ def has_been_notified_about(self, issue):
             return False
 
 
+class EventCount(models.Aggregate):
+    function = 'hll_cardinality'
+    template = 'hll_cardinality(hll_union_agg(%(expressions)s))'
+
+    def __init__(self, expression, **extra):
+        super().__init__(expression, output_field=models.FloatField(), **extra)
```

Comment:
In the code here:

https://github.com/django/django/blob/master/django/db/models/aggregates.py#L115

the `output_field` is listed as a class property and not passed in. Is there advantages to doing it the way you've done it here?

Reply:
I was following the example here: https://docs.djangoproject.com/en/2.0/ref/models/expressions/#creating-your-own-aggregate-functions

AFAICT what happens is that `output_field` is passed all the way down to the `BaseExpression` class, which assigns it to a [cached property](https://github.com/django/django/blob/3e36ecedffebb81e7e7bb8723f26fbd5056e9fe9/django/db/models/expressions.py#L261) if it is given. I'm not entirely sure how the class property comes into play.

I like having it as a class property better, so if that still passes the tests I'll change it to that.